src/tribler-gui/tribler_gui/event_request_manager.py

```diff
@@ -37,7 +37,7 @@ def __init__(self, api_port, api_key, error_handler):
         QNetworkAccessManager.__init__(self)
         url = QUrl("http://localhost:%d/events" % api_port)
         self.request = QNetworkRequest(url)
-        self.request.setRawHeader(b'X-Api-Key', api_key)
+        self.request.setRawHeader(b'X-Api-Key', api_key.encode('ascii'))
```

Comment:
**Question:** are you sure that 'ascii' is the correct encoding?

Reply:
Yes, because we construct `api_key` as `hexlify(os.urandom(16))`. It contains ASCII symbols only.